doc/python/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2016, EPFL/Blue Brain Project
+#                     Juan Hernando Vieites <juan.hernando@epfl.ch>
+#
+# This file is part of Brion <https://github.com/BlueBrain/Brion>
+#
+
+# Reusing the FindSphinx script from Pydoxine
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/Pydoxine/CMake)
+
+common_find_package(Sphinx)
```

Comment:
* FindSphinx.cmake should be in CMake/common then
* move common_find_package(Sphinx 1.3) to top-level
* then the default_docstrings.cpp can be removed?